src/Microsoft.Extensions.Configuration.UserSecrets/ConfigurationExtensions.cs

```diff
@@ -3,15 +3,19 @@
 
 using System;
 using System.IO;
+using System.Reflection;
 using Microsoft.Extensions.Configuration.UserSecrets;
+using Microsoft.Extensions.Configuration.UserSecrets.Internal;
 using Microsoft.Extensions.FileProviders;
 
 namespace Microsoft.Extensions.Configuration
 {
+    /// <summary>
+    /// Configuration extensions for adding user secrets configuration source.
+    /// </summary>
     public static class ConfigurationExtensions
     {
-        private const string Secrets_File_Name = "secrets.json";
-
+#if NET451 || NETSTANDARD1_5
```

Comment:
Yeah, this is a tricky one. Root problem is Assembly.GetEntryAssembly isn't in netstandard1.3, only netstandard1.5 and up. `AddUserSecrets()` needs this API to work right.

A couple of ideas for how to handle on netstandard1.3

- PlatformNotSupportedException 
- default to "project.json"
- remove from netstandard1.3 API surface.

In this draft I went for "remove", but on second though "PlatformNotSupportedException" seems more appropriate. Thoughts @Eilon?